src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -395,7 +398,7 @@ def _process_heatmap_tools(heatmap_tools: set[str],
         except AttributeError as e:
             raise ValueError(f'Please implement parsing method for: [{tool}]') from e
 
-        heatmap_meta_dict, heatmap_rows_list, heatmap_cell_details_list = parse_ops(
+        heatmap_meta_dict, heatmap_rows_list, heatmap_cell_details_list, bad_data_id = parse_ops(
```

Comment:
I would change `bad_data_id` to `bad_data_ids` throughout. Seems minor but accurate naming can help with comprehension

Reply:
Updated all bad_data_id to bad_data_ids!

Reply:
:+1: